templates/todo/api/js/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es2015",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
FWIW - I didn't end up needing this change locally and still had a good VS Code experience - I think.  I know you were running into some promise issues without this, but everything seemed to work for me end to end without it.  Could you maybe remove this and see if things still work?

Reply:
FWIIW, we use `es2020` for the VS Code extension. There is really no good reason to stick with old JS flavors if when the project is a node backend app.